src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp

```diff
@@ -621,6 +621,20 @@ class TDatabaseFeedManager final : public Observer<nlohmann::json&>
         return {};
     }
 
+    /**
+     * @brief Get OS CPE rules.
+     * @param osCpeRules returned information.
+     */
+    void getOsCpeRules(nlohmann::json& osCpeRules)
+    {
+        rocksdb::PinnableSlice ret;
```

Comment:
```suggestion
        rocksdb::PinnableSlice queryResult;
```

I think we can make a little improve in the variable name here

Reply:
Changed